### DISCUSSION TITLE
Getting Survey Survey Responses

### DISCUSSION THREAD

**Discussion by rachan2023** [albertkun/21S-ASIAAM-191A#152] [Help Desk] [answered]
I forgot to use the workflow to get responses from my microsoft form populated into my google sheet before I sent out my survey. Currently only new responses are populated in my google sheet as opposed to the ones that I got in class. What should I do?

**Comment by albertkun** [comment, marked as answer]
@rachan2023 you should do the following:
1. set-up your google sheets to do the geocoding when you get the data ([lab 5](https://github.com/albertkun/21S-ASIAAM-191A/blob/main/Week_5/Lab/readme.md))
2. new step: you need to create a new Microsoft flow to copy your Microsoft form data into Google sheets
3. run the new MS Flow
4. Geocode the data in the Google Sheets
5. Turn off the Microsoft Flow in the new step (#2).
6. Set-up the correct Microsoft flow to run when new data comes in from the form
